components/Search.jsx

```diff
@@ -207,7 +212,7 @@ class Search extends React.Component {
                         'searchbar-provider-selection-active': isEmpty(this.props.activeProviders)
                     });
                     allEntry = (
-                        <li className={itemClass} key="all" onClick={() => this.props.changeSearch("", null)}>{LocaleUtils.tr("search.all")}</li>
+                        <li className={itemClass} key="all" onClick={() => this.props.changeSearch("", null)} title="">{LocaleUtils.tr("search.all")}</li>
```

Comment:
What's the purpose of the empty `title` here and elsewhere?

Reply:
This is to hide the tooltip of the button in the selection list of search providers. An alternative could be the name of the individual search provider.